implants/lib/eldritch/src/sys/get_ip_impl.rs

```diff
@@ -55,6 +66,27 @@ fn create_dict_from_interface(starlark_heap: &Heap, interface: NetInterface) ->
     Ok(tmp_res)
 }
 
+#[cfg(not(target_os = "windows"))]
+fn create_dict_from_interface(starlark_heap: &Heap, interface: NetworkInterface) -> Result<Dict>{
```

Comment:
Hmm could we Abstract this type out or just move the function to take the three params. Don't love having two separate dictionary functions.